ioctl/newcmd/account/accountimport.go

```diff
@@ -0,0 +1,225 @@
+// Copyright (c) 2019 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package account
+
+import (
+	"fmt"
+	"github.com/iotexproject/iotex-core/ioctl"
+	"os"
+	"sigs.k8s.io/yaml"
```

Comment:
can use existing "gopkg.in/yaml.v2", let's not introduce a new dependency

Reply:
yes,   "sigs.k8s.io/yaml"  is deleted